src/reference/90-Developers-Guide/00.md

```diff
@@ -48,13 +48,13 @@ source compatibility of the build during 1.x.y.
 #### Binary compatibility of the plugins
 
 Binary compatibility ("bincompat") of the plugins means that a plugin
-that was published  for sbt version A works for another version B without recompilaition.
+that was published for sbt version A works for another version B without recompilation.
 sbt 0.13 has kept binary compatibility for 18 months as of March 2015.
-The stability here helps maintaining sbt plug ecosystem.
+The stability here helps maintain the sbt plugin ecosystem.
 Our goal for sbt 1.0 is to adopt Semantic Versioning, and maintain
 binary compatibility of the build during 1.x.y.
 
 From the development perspective, maintaining binary compatibility becomes
 an additional constraint that we need to worry about whenever we make changes.
-The of the problem is that sbt 0.13 does not distinguish between public API
-and internal impelmentation. Most things are open to plugins.
+The [what?] of the problem is that sbt 0.13 does not distinguish between public API
```

Comment:
Yep I assumed "meat" or "core" or something like that was the omitted word, but I also agree that it could probably be reworded better.